var/spack/repos/builtin/packages/wrf/package.py

```diff
@@ -381,6 +360,7 @@ def run_compile_script(self):
             error=str
         )
 
+        print(result_buf)
```

Comment:
@adamjstewart @alalazo Can you confirm that  this is the "Spack" accepted way to output the string buffer to the build logs (and also when verbose mode is called)?
I feel like it should be through some functionality in llnl.util.tty.

**Background**
In order to check that the build has built successfully we need to parse the output of the compilation for a completion message. But the current implementation would discard this buffer, so the user could not inspect the errors.

Reply:
Both tty and print are valid. Unfortunately there isn't a way to log to the screen when verbose is disabled. There have been several proposals to fix this but none of them were accepted.

Reply:
Thanks, so we will stay with this implementation